library/src/scripts/bootstrap.ts

```diff
@@ -17,6 +17,10 @@ import { _mountComponents } from "@library/utility/componentRegistry";
 import { blotCSS } from "@rich-editor/quill/components/blotStyles";
 import { bootstrapLocales } from "@library/locales/localeBootstrap";
 
+if (gdn.getMeta("Feature.UseFocusVisible.Enabled", true)) {
```

Comment:
I think this is named incorrectly.

```suggestion
if (gdn.getMeta("featureFlags.useNativeFocus.Enabled", true)) {
```

Also please add the feature and it's description to https://staff.vanillaforums.com/kb/articles/170-feature-flags